.github/workflows/test_dev_env.yml

```diff
@@ -18,4 +19,85 @@ jobs:
           docker-compose up -d
           sleep 30
           docker ps -a | (! grep Exited ) # Return a non-zero exit code if any of the containers are stopped
+  build:
+    name: Verify build
+    runs-on: ubuntu-latest
+    env:
+      POSTGRES_USER: daria
+      POSTGRES_PASSWORD: ""
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: daria
+          POSTGRES_DB: dcaf_case_management_test
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        # Add a health check
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Ruby setup
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1 # automagically reads from .ruby-version
+        with:
+          bundler-cache: true
+
+      # npm/yarn setup
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.2
+      - name: Find yarn cache location
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Cache yarn
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: yarn packages
+        run: |
```

Comment:
You can just do `run: yarn install`, the `|` is only if you need to do multiple commands

Reply:
good flag, thank you!